samples/basic/sys_heap/sample.yaml

```diff
@@ -0,0 +1,9 @@
+sample:
+  description: System heap sample showing how to use sys_heap functions
+  name: Basic system heap sample
+common:
+    integration_platforms:
+      - native_posix
+tests:
```

Comment:
Probably it's worth to add regex harness, so sample can be used as a test (by verifying console output).

See https://github.com/zephyrproject-rtos/zephyr/blob/v3.2.0/samples/philosophers/sample.yaml#L9-L16 as an example.

Reply:
Added